.github/workflows/report.yml

```diff
@@ -95,77 +148,134 @@ jobs:
+          ALL_CHANGED_FILES: ${{ needs.dependency-resolution.outputs.all_changed_files }}
+      - name: upload coverage report
+        if: ${{ inputs.use_simplecov_report == true }}
         uses: actions/upload-artifact@v2
         with:
-          name: tmp-coverage-report
-          path: coverage
-          if-no-files-found: error
-      - name: upload the results of the spec report
+          name: coverage-report-${{ github.run_id }}
+          path: ${{ inputs.simplecov_report_path }}
+          if-no-files-found: warn
+      - name: upload rspec report
+        uses: actions/upload-artifact@v2
+        with:
+          name: rspec-reports
+          path: rspec-reports
+          if-no-files-found: warn
+
+  test-in-parallel-with-mysql:
+    if: ${{ inputs.use_mysql == true && inputs.test_in_parallel == true }}
+    name: execute `make test`
+    needs: dependency-resolution
+    runs-on: ubuntu-${{ startsWith(inputs.mysql_version, '5') && '18.04' || 'latest' }}
+    services:
+      db:
+        image: mysql:${{ inputs.mysql_version }}
+        ports:
+          - 3306:3306
+        env:
+          MYSQL_ALLOW_EMPTY_PASSWORD: true
+        options: >-
+          --health-cmd "mysqladmin ping"
+          --health-interval 10s
+          --health-timeout 5s
+          --health-retries 5
+    strategy:
+      fail-fast: false
+      matrix:
+        node_count: [8]
+        node_index: [0, 1, 2, 3, 4, 5, 6, 7]
+    steps:
+      - uses: actions/checkout@v2
+      - uses: pinnacles/common-cicd-actions/.github/actions/setup-lang@v0.1.0
+        with:
+          use_ruby: 'true'
+          ruby_version: ${{ inputs.ruby_version }}
+      - uses: pinnacles/common-cicd-actions/.github/actions/download-rspec-reports@v0.5.3
+      - run: |
+          curl -L -o split-test https://github.com/mtsmfm/split-test/releases/download/v1.0.0/split-test-x86_64-unknown-linux-gnu
+          chmod +x split-test
+      - run: make test
+        env:
+          ACTION_TYPE: CI
+          RUBY_VERSION: ${{ inputs.ruby_version }}
+          NODE_COUNT: ${{ matrix.node_count }}
+          NODE_INDEX: ${{ matrix.node_index }}
+          JUNIT_REPORT_DIR_OLD: old-rspec-reports
+          JUNIT_REPORT_DIR_NEW: rspec-reports
+          GITHUB_TOKEN: ${{ secrets.github_pat }}
+          COVERAGE: ${{ inputs.use_simplecov_report }}
+          ALL_CHANGED_FILES: ${{ needs.dependency-resolution.outputs.all_changed_files }}
+      - name: upload coverage report
+        if: ${{ inputs.use_simplecov_report == true }}
+        uses: actions/upload-artifact@v2
+        with:
+          name: coverage-report-${{ github.run_id }}
+          path: ${{ inputs.simplecov_report_path }}
+          if-no-files-found: warn
+      - name: upload rspec report
         uses: actions/upload-artifact@v2
         with:
           name: rspec-reports
           path: rspec-reports
-          if-no-files-found: error
+          if-no-files-found: warn
```

Comment:
step to the next step even if the report didn't exist. It is possibly occur when test was skipped.